application/sandbox/serial/process/src/services/service.sessions.ts

```diff
@@ -98,8 +126,8 @@ class ServiceSessions {
                         }
                     }));
                 });
-            case ECommands.write:
-                return this._income_onWrite(message).then(() => {
+            case ECommands.send:
+                return this._income_onSend(message).then(() => {
```

Comment:
section `catch` in missed on promise

Reply:
There is a `catch `section is in line 138, or what do you exactly mean?